test/NNPDE_tests_gpu.jl

```diff
@@ -43,12 +43,15 @@ dx = 0.25; dy= 0.25; dt = 0.25
 # Neural network
 chain = FastChain(FastDense(3,16,Flux.σ),FastDense(16,16,Flux.σ),FastDense(16,1)) |>gpu
 
+initθ = initial_params(chain) |>gpu
+strategy = QuadratureTraining(algorithm=CubaCuhre(),reltol= 1e-5,abstol= 1e-5,maxiters=50)
```

Comment:
I'm not sure if the quadrature algorithms will sample on GPUs. I don't think they can.